src/layout/HeroSection/HeroSection.svelte

```diff
@@ -41,34 +34,24 @@
 		}, 500);
 	};
 
-	const updateDownloadSource = (value: number) =>
-		localStorage.setItem("downloadSource", value.toString());
+	const updateDownloadSource = (value: number) => localStorage.setItem("downloadSource", value.toString());
 
 	const changeDownloadSource = (downloadSource: DownloadSources, id: number) => {
 		updateDownloadSource(id);
 
-		switch (downloadSource) {
-			case "Winget (CLI)":
-				wingetDialogOpen = true;
-				break;
-			case "Microsoft Store":
-			case "Sideload Package":
-			case "Sideload Package (Preview)":
-				window.open(downloadSource === "Microsoft Store" ? getStoreUrl() : sideloadLink, "_blank");
-				break;
-		}
+		if (downloadSource !== "Winget (CLI)")
+			window.open(downloadSource === "Microsoft Store" ? getStoreUrl() : sideloadLink, "_blank");
+		else wingetDialogOpen = true;
 
 		isDownloadDropdownOpen = false;
 	};
 
 	onMount(async () => {
 		// Get the user's download preference
-		if (!localStorage.getItem("downloadSource"))
-			localStorage.setItem("downloadSource", "0");
-		currentDownloadSource =
-			parseInt(localStorage.getItem("downloadSource")) ?? 0;
+		if (!localStorage.getItem("downloadSource")) localStorage.setItem("downloadSource", "0");
```

Comment:
Same as above, prefer curly braces. Please don't put the if keyword and the body on the same line.